### PR TITLE
POLCT-157 Rally adapter support

### PR DESCRIPTION
**Pull request by rahulguna10**

### CODE REVIEW

**Review by sig-chakrava — APPROVED** [review]

File: io-template.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ IO:
         --jira.username=$JIRA_USERNAME 
         --jira.auth.token=$JIRA_AUTH_TOKEN 
         --jira.assignee=$JIRA_ASSIGNEE 
-        --jira.enable=$ENABLE_JIRA 
+        --jira.enable=$ENABLE_JIRA
```
Comment (sig-suhaslr) [inline io-template.yml:61]: Can we remove the extra space ?

File: prescription.sh [hunk 2]
```diff
@@ -183,6 +187,10 @@ function generateYML () {
 	    s~<<JIRA_ISSUES_QUERY>>~$jira_issues_query~g; \
 	    s~<<JIRA_USERNAME>>~$jira_username~g; \
 	    s~<<JIRA_AUTH_TOKEN>>~$jira_auth_token~g; \
+            s~<<RALLY_PROJECT_NAME>>~$rally_project_name~g; \
```
Comment (sig-suhaslr) [inline prescription.sh:190]: indent needs to be same as the above code

File: prescription.sh [hunk 3]
```diff
@@ -233,6 +241,10 @@ function generateYML () {
...
```
Comment (sig-suhaslr) [inline prescription.sh:244]: missing indentation here as well

**Review by sig-suhaslr — CHANGES_REQUESTED** [review]

File: prescription.sh (hunk 2 above)
Comment (rahulguna10) [reply prescription.sh:190]: indentation looks good on the branch -  here it shows that way
https://github.com/synopsys-sig/io-artifacts/blob/POLCT-157/prescription.sh

File: prescription.sh (hunk 3 above)
Comment (rahulguna10) [reply prescription.sh:244]: indentation looks good on the branch -  here it shows that way
https://github.com/synopsys-sig/io-artifacts/blob/POLCT-157/prescription.sh

File: prescription.sh (hunk 2 above)
Comment (sig-suhaslr) [reply prescription.sh:190]: Not sure might why it shows that way, but anyways thanks

File: prescription.sh (hunk 3 above)
Comment (sig-suhaslr) [reply prescription.sh:244]: Not sure might why it shows as missed indentation here, but if the indentation is correct in main code then its fine

**Review by sig-suhaslr — APPROVED** [review]
LGTM

**Pull request merged**